rnd/docker-compose.yml

```diff
@@ -6,11 +6,48 @@ services:
       POSTGRES_USER: agpt_user
       POSTGRES_PASSWORD: pass123
       POSTGRES_DB: agpt_local
-      PGUSER: 5432
     healthcheck:
       test: pg_isready -U $$POSTGRES_USER -d $$POSTGRES_DB
       interval: 10s
       timeout: 5s
       retries: 5
     ports:
       - "5432:5432"
+
+  redis:
+    image: redis:latest
+    command: redis-server --requirepass password
+    ports:
+      - "6379:6379"
+
+  rest_server:
+    build:
+      context: ../
+      dockerfile: rnd/autogpt_server/Dockerfile
+    depends_on:
+      - postgres
+      - redis
+    environment:
+      - DATABASE_URL=postgresql://agpt_user:pass123@postgres:5432/agpt_local
+      - REDIS_HOST=redis
+      - REDIS_PORT=6379
+      - REDIS_PASSWORD=password
+      - AUTH_ENABLED=false
+    ports:
+      - "8000:8000"
+
+  ws_server:
+    build:
+      context: ../
+      dockerfile: rnd/autogpt_server/Dockerfile.ws
+    depends_on:
+      - postgres
+      - redis
+    environment:
+      - DATABASE_URL=postgresql://agpt_user:pass123@postgres:5432/agpt_local
+      - REDIS_HOST=redis
+      - REDIS_PORT=6379
+      - REDIS_PASSWORD=password
+      - AUTH_ENABLED=false
+    ports:
+      - "8001:8001"
```

Comment:
Missing newline.

Tip: [NewLine VSCode extension](https://marketplace.cursorapi.com/items?itemName=chang196700.newline)
> Whenever you save a file, this extension will automatically add a blank line at its end, if there isn't one already. If you have multiple blank lines, only one will be kept.

Reply:
Add this to the suggested extensions @Pwuts